src/components/TodoApp.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useContext, useEffect } from 'react';
+import { Header } from './Header';
+import { Main } from './Main';
+import { Footer } from './Footer';
+import { TodoContext } from '../context/TodoContext';
+import { ErrorArea } from './ErrorArea';
+
+export const TodoApp: React.FC = () => {
+  const { todos, errorMessage, setErrorMessage } = useContext(TodoContext);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+
+    return () => clearTimeout(timer);
+  }, [errorMessage, setErrorMessage]);
```

Comment:
you can remove it, you have already implemented this logic insise `ErrorArea` component